sigh/lib/sigh/download_all.rb

```diff
@@ -1,13 +1,15 @@
 module Sigh
   class DownloadAll
     # Download all valid provisioning profiles
-    def download_all
+    def download_all(options, args)
+      download_xcode_profiles = options.download_xcode_profiles
```

Comment:
Ah, because these are the commander options, right? If so, can you rename the variable to `commander_options` to have it less confusing for people who know the rest of the code base? Thanks